lib/jekyll/tags/highlight.rb

```diff
@@ -23,7 +23,7 @@ def initialize(tag_name, markup, tokens)
 
             #{markup}
 
-            Valid syntax: highlight <lang> [linenos]
+            Valid syntax: highlight <lang> [linenos] [highlight_lines="3 4 5"] [highlight_line_class=class_name]
```

Comment:
Is there a strong reason to customize the highlight line class? Couldn't a user simply apply the styles they want to `.hll`?

Reply:
I was having the same thought today! I think we should remove this option for the time being. If we find it's a problem for people in the future, we can always add it. Feels like a premature optimisation right now. 

I'll fix.

Reply:
Removed in 8a202c50a6205fc0dda441ec8a52a686aac9a987